test/processor/datadir_test.go

```diff
@@ -63,6 +63,9 @@ func RunDataDirTest(t *testing.T, testDir string, binary string) {
 		}
 	}
 
+	// Gzip files for easier definition
+	gzipAllInDir(t, dataDir+"/in")
+
```

Comment:
V datadir testoch je mozne v `out` - expected dir vytvorit subory s ext `.ungzipped`.
Pouziva sa to ako procesor vytvara gzip subory, tak aby v git repe boli v dekodovanom formate.

V tomto commite som to pridal aj do `in` ale opacne, ze sa tieto subory pred testom prevedu do `gzip`.